ui/src/literature/components/DOILinkAction.jsx

```diff
@@ -1,30 +1,73 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { List } from 'immutable';
+import { Button, Menu } from 'antd';
 
 import ListItemAction from '../../common/components/ListItemAction';
 import EventTracker from '../../common/components/EventTracker';
+import DropdownMenu from '../../common/components/DropdownMenu';
+import IconText from '../../common/components/IconText';
+import ExternalLink from '../../common/components/ExternalLink';
 
 class DOILinkAction extends Component {
+  static renderLinkForDoi(doi) {
+    return `//doi.org/${doi.get('value')}`;
+  }
+
+  static renderDoi(doi) {
+    const material = doi.get('material');
+    return (
+      <span>
+        <span>{doi.get('value')}</span>
+        {material && <span> ({material})</span>}
+      </span>
+    );
+  }
+
+  renderDoiLinks() {
+    const { dois } = this.props;
+    return dois.map(doi => (
+      <Menu.Item key={doi.get('value')}>
+        <ExternalLink href={DOILinkAction.renderLinkForDoi(doi)}>
+          {DOILinkAction.renderDoi(doi)}
+        </ExternalLink>
+      </Menu.Item>
+    ));
+  }
+
   render() {
-    const { doi } = this.props;
-    const href = `//doi.org/${doi}`;
+    const { dois } = this.props;
+    const text = 'DOI';
+    const type = 'link';
     return (
-      <EventTracker eventId="DoiLink">
-        <ListItemAction
-          iconType="link"
-          text="DOI"
-          link={{
-            href,
-            target: '_blank',
-          }}
-        />
-      </EventTracker>
+      <ListItemAction>
+        <EventTracker eventId="DoiLink" eventPropName="onClick">
+          {dois.size > 1 ? (
+            <DropdownMenu
```

Comment:
I would separate these two parts into `renderMultipleDOIDropdown`and `renderDOI`